sdks/common/fn-api/src/main/proto/beam_fn_api.proto

```diff
@@ -59,7 +59,7 @@ message Target {
   }
 
   // (Required) The id of the PrimitiveTransform which is the target.
-  int64 primitive_transform_reference = 1;
```

Comment:
Fine by me, but the low latency and blanket nature of your response makes me think the performance details are not well understood.

This will likely have some cost in loss of reuse between Runner PI and Fn API, and some burden on runners to convert.